src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { uploadManifest } from "./manifest";
+import { generateManifest, Action } from "./environment";
+import { uploadAction, uploadStream } from "./upload";
+import { createReadStream } from "fs";
+import { getConfig } from "./settings";
+
+export const deploy = async () => {
+  const conf = await getConfig();
+
+  const projectName = conf.projectName;
+  const vcsURL = conf.vcsURL || "https://gu.com";
+  const actions: Action[] = conf.actions;
+
+  const manifest = generateManifest(projectName, vcsURL);
+  console.log(manifest);
```

Comment:
is this still needed? I guess so as it has useful info, maybe it needs some context?